.rspec

```diff
@@ -1,2 +1,4 @@
 --color
 --require spec_helper
+--format d
```

Comment:
`documentation` instead of `d`. It is always better to use full names for arguments whenever possible. You can use abbreviations in inline usecases.